test/functional/test_runner.py

```diff
@@ -225,6 +225,7 @@
     'feature_nulldummy.py --descriptors',
     'mempool_accept.py',
     'mempool_expiry.py',
+    'mempool_fee_histogram.py',
```

Comment:
The new test file has incorrect permissions:
```
-rwxr-xr-x 1   5133 Mar 17 15:30 mempool_expiry.py*
-rw-r--r-- 1   1716 Mar 21 11:56 mempool_fee_histogram.py
-rwxr-xr-x 1   3494 Mar  1 11:53 mempool_limit.py*
```
I had to run `sudo chmod 755 test/functional/mempool_fee_histogram.py` to be able to call the test directly, e.g. with `test/functional/mempool_fee_histogram.py`

Reply:
I fixed this. Thanks!